TYPESCRIPT.md

```diff
@@ -116,10 +117,34 @@ getEntries<ExampleEntrySkeleton>({
 })
 ```
 
-#### Limitation
+#### Limitations
 
-Due to the limitation with recursive types, we can only calculate keys on the root level of your content type.
+- Due to the limitation with recursive types, we can only calculate keys on the root level of your content type.
 There is currently no way to calculate keys for nested (recursive) content types.
+- To limit the complexity of query types we use a simple type definition for [search on references](https://contentful.atlassian.net/wiki/spaces/PROD/pages/4169466000/Known+limitations+of+contentful.js+TypeScript+support#:~:text=search%20on%20references). 
```

Comment:
This refers to the same limitation as the first point. Let’s remove the first as that is less accurate (it’s possible to calculate the keys, just not desirable if you want to actually use the types).

Reply:
I was aware that they were very similar just needed your input on which one is better suited, thank you!